src/main/java/io/lettuce/core/XReadArgs.java

```diff
@@ -1,11 +1,11 @@
 package io.lettuce.core;
 
-import java.time.Duration;
-
 import io.lettuce.core.internal.LettuceAssert;
 import io.lettuce.core.protocol.CommandArgs;
 import io.lettuce.core.protocol.CommandKeyword;
 
+import java.time.Duration;
```

Comment:
nit: not needed

Reply:
Its used in  `public static XReadArgs block(Duration timeout)` and `public XReadArgs block(Duration timeout)` as far as I can see